client/src/pages/LessonHome/LessonHome.js

```diff
@@ -65,10 +68,12 @@ const LessonHome = ({ navigation }) => {
         for (let i = 0; i < lessonData.vocab.length; i += 1) {
           const item = lessonData.vocab[i]
 
+          // This is where we will check cache once cache is created
```

Comment:
You can probably make the comment a bit more concise and make it say something like "checks cache on creation"